scripts/master.sh

```diff
@@ -1,19 +1,15 @@
 #!/usr/bin/bash
 set -eu
 
-KUBERNETES_VERSION=${KUBERNETES_VERSION:-}
-CORE_DNS=${CORE_DNS:-}
-
-echo "
-Package: kubectl
-Pin: version ${KUBERNETES_VERSION}-*
-Pin-Priority: 1000
-" > /etc/apt/preferences.d/kubectl
-
-apt-get install -qq -y kubectl
-
+PRIVATE_IP=""
+while [ -z $PRIVATE_IP ]
+do
+  sleep 1
+  PRIVATE_IP=$(ifconfig | grep -A 1 'ens10' | tail -1 | awk '{print $2}'| cut -d ':' -f 2)
+done
+PUBLIC_IP=$(ifconfig | grep -A 1 'eth0' | tail -1 | awk '{print $2}'| cut -d ':' -f 2)
 # Initialize Cluster
-kubeadm init --feature-gates CoreDNS="$CORE_DNS"
+kubeadm init --apiserver-advertise-address $PRIVATE_IP --pod-network-cidr=192.168.0.0/16 --apiserver-cert-extra-sans $PUBLIC_IP
```

Comment:
Don't drop the support of feature gates. Because in future you also want have the option to do this. I think it would be better to have this option it could be described like this:

```console
if [[ -n "$FEATURE_GATES ]]; then
kubeadm init --apiserver-advertise-address $PRIVATE_IP --pod-network-cidr=192.168.0.0/16 --apiserver-cert-extra-sans $PUBLIC_IP --feature-gates $FEATURE_GATES
fi
```